tests/fixtures/test_brief.py

```diff
@@ -1187,3 +1193,506 @@ def test_rfx_validate_closed_at():
+                                                         atm_brief, get_day_count, atm_data):
+    res = client.post('/2/login', data=json.dumps({
+        'emailAddress': 'me@digital.gov.au', 'password': 'test'
+    }), content_type='application/json')
+    assert res.status_code == 200
+
+    data = atm_data
+    data['publish'] = True
+    data['closedAt'] = pendulum.today().add(days=get_day_count).format('%Y-%m-%d')
+
+    res = client.patch('/2/brief/1', content_type='application/json', data=json.dumps(data))
+    assert res.status_code == 200
+    response = json.loads(res.data)
+    assert response['closedAt'] == pendulum.today().add(days=get_day_count).format('%Y-%m-%d')
+    assert response['dates']['questions_closing_date'] == (
+        pendulum.instance(workday(pendulum.today(), 5)).format('%Y-%m-%d')
+    )
+
+
+def test_atm_brief_create_failure_as_seller(client, supplier_user):
+    res = client.post('/2/login', data=json.dumps({
+        'emailAddress': 'j@examplecompany.biz', 'password': 'testpassword'
+    }), content_type='application/json')
+    assert res.status_code == 200
+
+    res = client.post('/2/brief/atm', content_type='application/json')
+    assert res.status_code == 403
+
+
+@mock.patch('app.tasks.publish_tasks.brief')
+def test_atm_brief_update_success(brief, client, buyer_user, atm_brief):
+    res = client.post('/2/login', data=json.dumps({
+        'emailAddress': 'me@digital.gov.au', 'password': 'test'
+    }), content_type='application/json')
+    assert res.status_code == 200
+
+    res = client.patch('/2/brief/1', content_type='application/json', data=json.dumps({
+        'closedAt': pendulum.today().add(weeks=2).format('%Y-%m-%d')
+    }))
+    assert res.status_code == 200
+    response = json.loads(res.data)
+    assert response['closedAt'] == pendulum.today().add(weeks=2).format('%Y-%m-%d')
+
+
+@mock.patch('app.tasks.publish_tasks.brief')
+def test_atm_brief_update_failure_closing_date_invalid(brief, client, buyer_user, atm_brief):
+    res = client.post('/2/login', data=json.dumps({
+        'emailAddress': 'me@digital.gov.au', 'password': 'test'
+    }), content_type='application/json')
+    assert res.status_code == 200
+
+    res = client.patch('/2/brief/1', content_type='application/json', data=json.dumps({
+        'publish': True,
+        'closedAt': 'baddate'
+    }))
+    assert res.status_code == 400
+
+
+@mock.patch('app.tasks.publish_tasks.brief')
+def test_atm_brief_update_failure_unknown_property(brief, client, buyer_user, atm_brief):
+    res = client.post('/2/login', data=json.dumps({
+        'emailAddress': 'me@digital.gov.au', 'password': 'test'
+    }), content_type='application/json')
+    assert res.status_code == 200
+
+    res = client.patch('/2/brief/1', content_type='application/json', data=json.dumps({
+        'publish': True,
+        'xxx': 'yyy'
+    }))
+    assert res.status_code == 400
+
+
+def test_atm_validate_location():
```

Comment:
Does it make sense to split these validation tests up so they're testing one thing at a time?

`test_atm_validates_location_when_xyz`